pubsub/google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -242,12 +242,17 @@ def publish(self, topic, data, **attrs):
             instance that conforms to Python Standard library's
             :class:`~concurrent.futures.Future` interface (but not an
             instance of that class).
+
+        Raises:
+            ValueError:
```

Comment:
```suggestion
            RuntimeError:
```
(now that we changed the code) :)